robottelo/constants/__init__.py

```diff
@@ -1480,7 +1480,7 @@ class Colored(Box):
 OSCAP_DEFAULT_CONTENT = {
     'rhel6_content': 'Red Hat rhel6 default content',
     'rhel7_content': 'Red Hat rhel7 default content',
-    'jre_content': 'Red Hat jre default content',
+    # 'jre_content': 'Red Hat jre default content',
```

Comment:
Is it good to remove from file? @reviewers

Reply:
@vijaysawant yes, it is not supported anymore so we can remove this, instead of commenting

Reply:
I saw some older constants are kept as it is by commenting, so didn't remove that one.